src/common/vars/is-linux.injectable.ts

```diff
@@ -3,11 +3,10 @@
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
 import { getInjectable } from "@ogre-tools/injectable";
-import { isLinux } from "../vars";
 
 const isLinuxInjectable = getInjectable({
   id: "is-linux",
-  instantiate: () => isLinux,
+  instantiate: () => process.platform === "linux",
```

Comment:
I think that `process.platform` should be an `injectable` itself so that these are consistent while in testing while only having to override a single injectable.

Reply:
Sure. We've been done that when we have multiple usages for e.g `process`.

Reply:
We already have multiple uses of `process`, including of `process.env`

Reply:
Sure. :)

Reply:
There is now `platformInjectable` which is overridden for same effect. This allowed us to get rid of `causesSideEffects` in `isLinux`, `isWindows` and `isMac`.

Needs to said:

All `isLinux`, `isWindows`, `isMac` and such are breach of Open Closed Principle and thus live on borrowed time. This is topic for the future.